notebooks/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>Based on this it does look that the default condition (0.3) may be a bit high (given that this one looks clear and is only 0.32), but I guess it's better to err to the higher side</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>